addon/styles/addon.scss

```diff
@@ -132,3 +132,9 @@
     }
   }
 }
+
+.ember-tooltip-or-popover-target-short-circuit {
```

Comment:
This change is [documented here](https://github.com/sir-dunxalot/ember-tooltips/pull/160/files#diff-04c6e90faac2675aa89e2176d2eec7d8R975).